step-functions/oauth-token-generator.asl.json

```diff
@@ -93,13 +35,8 @@
               "Parameters": {
                 "FunctionName": "${BearerTokenFunctionArn}",
                 "Payload": {
-                  "totp.$": "$.totpCode.value",
-                  "clientSecret": {
-                    "value.$": "$.clientSecret.SecretString"
-                  },
-                  "clientId": {
-                    "value.$": "$.clientId.SecretString"
-                  },
+                  "stackName.$": "$.StackName.value",
```

Comment:
Can probably rename this step to better reflect what is happening here now?

Reply:
What do you suggest as a name? This is just passing the stack name as a parameter to the Lambda.